pkg/validators/middlewares.go

```diff
@@ -1,51 +0,0 @@
-package validators
```

Comment:
Realized we are wiping the concept of middlewares completly. Is this intended?

Reply:
For the moment yes (See the next comment).